features/users/user_rename.feature

```diff
@@ -110,3 +110,30 @@ Feature:
     When I follow "Pseuds (2)"
       Then I should see "Edit oldusername"
       And I should see "Edit newusername"
+      
+  Scenario: Changing username updates search results (bug AO3-3468)
+    Given I have no users
+      And I am logged in as "oldusername" with password "password"
+      And I post a work "Epic story"
+    When I visit the change username page for oldusername
+      And I fill in "New user name" with "newusername"
+      And I fill in "Password" with "password"
+      And I press "Change User Name"
+    Then I should get confirmation that I changed my username
+    When I am on the the works page
+    Then I should see "newusername"
+      And I should see "Epic story"
+      And I should not see "oldusername"
+    # doesn't work if search indexes are not updated
```

Comment:
It's a note about unfortunate but expected behaviour at this point - would a clearer phrasing be "Has old name until indexes are updated"?